### DISCUSSION TITLE
Filecoin Vietnam Market Support

### DISCUSSION THREAD

**Discussion by dickytran** [filecoin-project/community#384] [Market]
Dear Filecoin Team,

We represent a large community in Vietnam market _ The largest Filecoin community in Vietnam.
Over the past time, we have clearly understood the long-term vision and core values   of Filecoin, so we have actively created channels, and connected the community of people who are interested in Filecoin, and IPFS in Vietnam.
With this Email, we hope to receive official support from Filecoin, like the supporting for the Korean market.
Our community in Vietnam fully believes in the value that Filecoin and IPFS will bring to the future industry of mankind. Besides the Channels about Filecoin, we also continuously organize Zoom sessions to train and share the values   and vision of Filecoin for the Vietnamese community.
Here are a few channels that we have created and operated recently:

https://filecoinvietnam.com/

https://t.me/filecoinvietnam_mining

https://t.me/filecoinvietnam

https://www.facebook.com/groups/filecoinvietnams/

Please check the attachment, to see the number of members in our community!
We are looking forward to receiving feedback, as well as receiving official support from Filecoin.

Sincere thanks!
THE BIGGEST FILECOIN COMMUNITY IN VIETNAM

**Comment by Joss-Hua** [comment]
It is exciting that there are more and more ecological participants.  :wave:

**Comment by dickytran** [comment]
We're so glad to be with you, Guys!!!

Vào Th 4, 17 thg 11, 2021 vào lúc 15:16 Joss ***@***.***>
đã viết:

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
let's keep it in this thread.

can you please stop spamming all available chans with the same text? you posted this dozens of times on slack and now we have 3 threads there in the discussion forums. 

thanks

**Comment by dickytran** [comment]
Hi Bro,
So sorry about that but its a 2 days ago, now i will delete 2 of them

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
don't worry - i already deleted them :P

**Comment by dickytran** [comment]
Yeah, thanks bro ;)


Vào Th 5, 18 thg 11, 2021 vào lúc 05:50 Patrick ***@***.***>
đã viết:

**Comment by gdragonHH** [comment]
硬件算力出海，专业技术团队 独立节点，欢迎大家加V了解：Hmcxxxxx
